### PR TITLE
Refactor session feedback layout

### PR DESCRIPTION
**Pull request by konifar**
Reduce over 100 rows of `activity_session_feedback.xml`.

### CODE REVIEW

File: app/src/main/res/layout/activity_session_feedback.xml [hunk 1]
```diff
@@ -13,8 +13,9 @@
             android:id="@+id/toolbar"
             android:layout_width="match_parent"
             android:layout_height="?attr/actionBarSize"
-            android:layout_marginBottom="@dimen/spacing_small"
             android:background="?attr/colorPrimary"
+            android:elevation="@dimen/elevation"
+            app:elevation="@dimen/elevation"
```
Comment (konifar) [inline app/src/main/res/layout/activity_session_feedback.xml]: Toolbar shadow.

File: app/src/main/res/layout/activity_session_feedback.xml [hunk 2]
```diff
@@ -25,193 +26,106 @@
...
-                            android:layout_height="wrap_content"
-                            android:layout_marginEnd="@dimen/spacing_small"
-                            android:layout_marginLeft="@dimen/spacing_small"
-                            android:layout_marginRight="@dimen/spacing_small"
-                            android:layout_marginStart="@dimen/spacing_small"
-                            android:maxHeight="?android:listPreferredItemHeightSmall"
-                            android:minHeight="?android:listPreferredItemHeightSmall"
-                            android:paddingTop="@dimen/spacing_small" />
-
-                        <FrameLayout
```
Comment (konifar) [inline app/src/main/res/layout/activity_session_feedback.xml]: If we use RelativeLayout in parent, it is not necessary to use `FrameLayout`.

**Comment by konifar** [comment]
@hotchemi Refactored. Could you review it?

**Comment by hotchemi** [comment]
Nice!

**Pull request merged**